package.json

```diff
@@ -16,17 +16,19 @@
   "repository": "github:LD4P/sinopia_indexing_pipeline",
   "scripts": {
```

Comment:
Hi @mjgiarlo, when reviewing this yesterday I thought it might be worthwhile to add two NPM scripts for the `reindex` and `search`. I created a branch https://github.com/LD4P/sinopia_indexing_pipeline/tree/add-npm-commands that adds these to the `package.json` based on your PR that you could merge into this PR or I'm happy to create a new PR after this one is merged.

Reply:
@jermnelson OK. Can you say a bit more about what it adds? 

I'm not opposed to the change though I wonder if it introduces ambiguity re: the patterns that we use going forward, specifically answering the question of "should this command-line thingie be an `npm` script or a `bin` command?"

What I *thought* was the distinction was that `npm` scripts were developer-facing and `bin` commands are made available to all clients of the package (thus, we can invoke them anywhere once we have the container, *e.g.*, in CircleCI and in AWS).

If that **is** the relevant distinction, that suggests that the `reindex` operation should remain a `bin` command and `search` should move out of `bin` and into an `npm` script.

Your head is more in JavaScriptland than mine is at this point, so I'm happy to defer ultimately to your expertise. :smile: cc: @LD4P/sinopia, so they can see the ensuing discussion and the pattern we settle on. Thanks, @jermnelson!

Reply:
> @jermnelson OK. Can you say a bit more about what it adds?
> 
> I'm not opposed to the change though I wonder if it introduces ambiguity re: the patterns that we use going forward, specifically answering the question of "should this command-line thingie be an `npm` script or a `bin` command?"
> 
> What I _thought_ was the distinction was that `npm` scripts were developer-facing and `bin` commands are made available to all clients of the package (thus, we can invoke them anywhere once we have the container, _e.g._, in CircleCI and in AWS).
> 
> If that **is** the relevant distinction, that suggests that the `reindex` operation should remain a `bin` command and `search` should move out of `bin` and into an `npm` script.
> 
I agree
> Your head is more in JavaScriptland than mine is at this point, so I'm happy to defer ultimately to your expertise. 😄 cc: @LD4P/sinopia, so they can see the ensuing discussion and the pattern we settle on. Thanks, @jermnelson!
> 
Hi @mjgiarlo, I would say my head is only selective areas of JavaScriptland.

> P.S. Once we settle this, I'm 💯 happy to make any changes in this branch. Shouldn't take but a couple minutes.